src/main/webapp/WEB-INF/views/admin/project/common-js.jsp

```diff
@@ -24,7 +24,7 @@ var com_evt = {
 		
 		$('.btnCommentHistory').on('click', function(e){
 			e.preventDefault();
-			openCommentHistory("prj", "${project.prjId}");
+            openCommentHistory('<c:url value="/comment/popup/prj/${project.prjId}"/>');

```

Comment:
Dear @su-ram 
Indentation is required (change 12 space to 3 tab)

Reply:
Thanks for the find tips. I applied it. 👍